Zend/zend_exceptions.stub.php

```diff
@@ -22,6 +22,9 @@ function getPrevious();
 
     /** @return string */
     function getTraceAsString();
+
+    /** @return string */
+    function __toString();
```

Comment:
Can stubs not use the return type syntax?

Reply:
See this thread starting from here: https://github.com/php/php-src/pull/5083#discussion_r365906404